src/region/RegionCreator.tsx

```diff
@@ -116,10 +120,23 @@ export default function RegionCreator({ className, onAbort, onStart, onStop }: P
     };
     const updateDraw = (x: number, y: number): void => {
         const [x2, y2] = getPosition(x, y);
+        const { current: x1 } = positionX1Ref;
+        const { current: y1 } = positionY1Ref;
+
+        // Suppress the creation of a small region if the intention of the user is to click on the document
+        const isValid = Math.abs(x2 - (x1 ?? 0)) >= MIN_SIZE || Math.abs(y2 - (y1 ?? 0)) >= MIN_SIZE;
```

Comment:
Let's still move this out so we can only invoke it if `positionX2Ref.current` is null.